src/transform/src/analysis/equivalences.rs

```diff
@@ -587,6 +587,32 @@ impl EquivalenceClasses {
             new_equivalences.push(class);
         }
 
+        // If we see records formed from other expressions, we can equate the expressions with
+        // accessors applied to the class of the record former. In `minimize_once` we reduce by
+        // equivalence class representative before we perform expression simplification, so we
+        // shoud be able to just use the expression former, rather than find its representative.
+        // The risk, potentially, is that we would apply accessors to the record former and then
+        // just simplify it away learning nothing.
+        for class in self.classes.iter() {
+            for expr in class.iter() {
+                // Record-forming expressions can their accessors and their members.
```

Comment:
`// Record-forming expressions can ? their accessors and their members.`

Word missing.

Reply:
Yup